### PR TITLE
[Edge] Develop content under Edge use case section

### PR DESCRIPTION
**Pull request by ASawwaf**
[Edge] Draft the initial content for Edge #1415 

•	Terminologies 
•	Objective ( High level Objective ) 
•	Scope ( focus on Edge Scope , in/out scope , Next Slide ) 
•	CNTT Edge Principle & Strategy 
•	CNTT Edge roadmap 
•	Use cases ( that CNTT Edge will focus )  
•	SDO ( Engagement with other SDOs)

### CODE REVIEW

File: doc/tech/usecases.md [hunk 1]
```diff
@@ -11,3 +11,12 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+Content 
+•	Objective ( High level Objective )
```
Comment (CsatariGergely) [inline doc/tech/usecases.md:16]: Can you please use markdown syntax (- or *) for bulleted list?

File: doc/tech/usecases.md [hunk 2]
```diff
@@ -11,3 +11,12 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+Content 
+•	Objective ( High level Objective ) 
+•	Scope ( focus on Edge Scope , in/out scope , Next Slide )
```
Comment (CsatariGergely) [inline doc/tech/usecases.md:17]: Next slide?

File: doc/tech/usecases.md (hunk 1 above)
Comment (ASawwaf) [reply doc/tech/usecases.md:16]: sure I will  :)

File: doc/tech/usecases.md (hunk 2 above)
Comment (ASawwaf) [reply doc/tech/usecases.md]: it is Just draft point that we need to work on it

Comment (CsatariGergely) [reply doc/tech/usecases.md]: Still I think we should no have next slide here unless we refer to the slideset.

Comment (ASawwaf) [reply doc/tech/usecases.md]: Thanks, I already removed it after your first comment :)

**Pull request merged**